src/index.css

```diff
@@ -47,22 +47,30 @@ body:before {
 }
 
 /*
-Focus ring styling is disabled by default and instead we are using the :focus-ring polyfill. The
-:focus-ring pseudo selector is a future browser standard that differentiates between mouse
-and keyboard usage when determining whether to draw a focus ring. Usually we only want a
-focus ring in the latter case.
+  This will hide the focus indicator if the element receives focus via the mouse,
+  but it will still show up on keyboard focus.
 */
-
-*:focus {
+.js-focus-visible :focus:not(.focus-visible) {
   outline: none;
 }
 
-*.focus-ring {
+/*
+  We use a stronger and consistent focus indicator when an element receives focus via
+  keyboard.
+*/
+.js-focus-visible .focus-visible {
+  outline: none;
   border-radius: 4px;
   box-shadow: inset 0px 0px 0px 2px #4469E1;
```

Comment:
Could you move the #4469E1 color string into `Colors.js`?

Reply:
This is a css file and we can use the global colors from `Colors.js` here once we are using the globals mechanism of StyledComponents probably.

Reply:
Gna, of course :) Then in the merged other version.